schedule/requirements.txt

```diff
@@ -3,7 +3,7 @@ argparse==1.1
 django-coverage==1.2.4
 django-nose==1.1
 nose==1.3.0
-python-dateutil==2.1
+altschool-python-dateutil==2.1
```

Comment:
so i think you could simply use a git link to the source as we do elsewhere.  that would install dateutil from the correct location without renaming things